src/server.js

```diff
@@ -64,7 +64,7 @@ export function _render (self, ProvideContext) {
       const __html = renderToStaticMarkup(elementWithProviders)
       return <div dangerouslySetInnerHTML={{__html}} />
     } catch (e) {
-      return <div>{self.props.fallBack()}</div>
+      return <>{self.props.fallBack()}</>
```

Comment:
I think, the Fragmet syntax is excess here:) fallBack'll return some jsx elements already

Reply:
If the server's fallback gets an exception's error  in arguments, it would be cool too;)
```
...
catch (error) {
  return self.props.fallBack({ error })
}
```

Reply:
This is going to be a problem when React tries to hydrate the DOM. If the DOM structure doesn't match what would normally be rendered, then it can result in zombie elements which do not get re-rendered. The fallback content isn't going to be the same as whatever we expected to render, but removing that div even with no fallback will be something that hydrate handles badly.